File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/Go/s545439898_comments_removed.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	countStr, _ := reader.ReadString('\n')
	count, _ := strconv.Atoi(strings.TrimSpace(countStr))

	givingNumbers := make([]int, count)
	for i := 0; i < count; i++ {
		numStr, _ := reader.ReadString(' ')
		num, _ := strconv.Atoi(strings.TrimSpace(numStr))
		givingNumbers[i] = num
	}

	for i := count - 1; i >= 0; i-- {
		fmt.Print(givingNumbers[i])
		if i > 0 {
			fmt.Print(" ")
		}
	}
	fmt.Println()
}
// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/C++/s545439898_comments_removed.cpp
#include <iostream>
#include <vector>

int main() {
    std::vector<int> givingNumbers;
    int count;
    std::cin >> count;
    givingNumbers.resize(count);

    for(int i = 0; i < count; i++) {
        std::cin >> givingNumbers[i];
    }

    for(int i = count - 1; i >= 0; i--) {
        std::cout << givingNumbers[i];
        if(i > 0) {
            std::cout << " ";
        }
    }

    std::cout << std::endl;

    return 0;
}
// End of Code
